helper_methods.rb

```diff
@@ -0,0 +1,121 @@
+module HelperMethods
+
+  def register_user
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+    @login = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys(@login)
+    @driver.find_element(:id, 'user_lastname').send_keys(@login)
+    @driver.find_element(:id, 'user_mail').send_keys(@login + '@q.com')
+    @driver.find_element(:name, 'commit').click
+  end
+
+  def register_user1
+    @driver.navigate.to 'http://demo.redmine.org'
+    @driver.find_element(:class, 'register').click
+    @wait.until {@driver.find_element(:id, 'user_login').displayed?}
+    @login1 = 'login'+rand(99999).to_s
+    @password = 'Qwerty11'
+    @driver.find_element(:id, 'user_login').send_keys(@login1)
+    @driver.find_element(:id, 'user_password').send_keys(@password)
+    @driver.find_element(:id, 'user_password_confirmation').send_keys(@password)
+    @driver.find_element(:id, 'user_firstname').send_keys(@login1)
+    @driver.find_element(:id, 'user_lastname').send_keys(@login1)
+    @driver.find_element(:id, 'user_mail').send_keys(@login1 + '@q.com')
+    @driver.find_element(:name, 'commit').click
+  end
+
+  def login_user
+    if @driver.find_element(:class, 'logout').displayed?
```

Comment:
I don't want to this instead of wait.until. I just want to logout if already logged in. Can I leave it as is?